winterfell/services/claimService.js

```diff
@@ -5,19 +5,11 @@ var Claim = require('./../models/claim');
 
 function ClaimService(app) {
   this.app = app;
-};
+}
 
 module.exports = ClaimService;
 module.exports.createNewClaim = function(userId, callback) {
-  return Claim.findOne({ userId: userId, state: Claim.State.INCOMPLETE }).exec(function(err, fileClaim) {
```

Comment:
Hmm ok I have an idea, maybe we can rename this method createOrFindExistingClaim and then have the `new claim` button read `finish in progress claim` if the user has a started claim.